packages/cli-common/resources/templates/template-project/admin/index.sass

```diff
@@ -0,0 +1 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap')
```

Comment:
Do we use all weights in admin?

Reply:
We use some of them, not all, but by reducing we might break styles for the client that might not expect we did not include some of the weights. If we add just some, we need to document it somewhere, I guess.